Tests/TestingApps/PaywallsTester/PaywallsTester/Config/SamplePaywalls.swift

```diff
@@ -633,25 +633,335 @@ private extension SamplePaywallLoader {
 
     static func createFakePaywallComponentsData(components: [PaywallComponent], localization: [LocaleID: LocalizationDictionary]) -> PaywallComponentsData {
         PaywallComponentsData(templateName: "Component Sample",
-                               assetBaseURL: URL(string:"https://assets.pawwalls.com/")!,
-                               componentsConfig: PaywallComponentsData.ComponentsConfig(components: components),
+                              assetBaseURL: URL(string:"https://assets.pawwalls.com/")!,
+                              componentsConfig: PaywallComponentsData.ComponentsConfig(components: components),
                               componentsLocalizations: localization,
-                               revision: 0,
-                               defaultLocaleIdentifier: "en_US")
+                              revision: 0,
+                              defaultLocaleIdentifier: "en_US")
     }
 
+    internal static func packagesComponents(slide: Int) -> PaywallComponentsData {
+        return createFakePaywallComponentsData(components: packagesSample(slide: slide), localization: allStrings())
+    }
+
+//    internal static var packagesComponents: PaywallComponentsData {
+//        return createFakePaywallComponentsData(components: packagesSample, localization: allStrings())
+//    }
```

Comment:
feels like this was left over?

Reply:
Thanks, the paywalls sample file is in a high degree of flux, I'll tidy these up.